### PR TITLE
Feature/fix including realm with comma

### PR DESCRIPTION
**Pull request by SamvelRaja**
closes the issue https://github.com/postmanlabs/postman-app-support/issues/2522

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-oauth1",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (czardoz) [inline package.json]: I'll do this during release, can you revert this change? @SamvelRaja

File: index.js [hunk 2]
```diff
@@ -290,10 +290,11 @@ OAuth.setProperties(OAuth, // utility functions
         ,
         /** Construct the value of the Authorization header for an HTTP request. */
         getAuthorizationHeader: function getAuthorizationHeader(realm, parameters) {
-            var header = 'OAuth ';
+            var header = 'OAuth ',
+                headerParams = [];
 
             if (realm && realm.trim()) {
-                header = header + 'realm="' + OAuth.percentEncode(realm) + '"';
+                headerParams.push(['realm', OAuth.percentEncode(realm)].join('='));
```
Comment (czardoz) [inline index.js]: the percent encoded realm should be in double quotes

File: index.js [hunk 3]
```diff
@@ -310,10 +311,10 @@ OAuth.setProperties(OAuth, // utility functions
                 }
 
                 if (name.indexOf('oauth_') == 0) {
-                    header = header + OAuth.percentEncode(name) + '="' + OAuth.percentEncode(value) + '",';
+                    headerParams.push([OAuth.percentEncode(name), OAuth.percentEncode(value)].join('='));
```
Comment (czardoz) [inline index.js]: The value should be in quotes

**Comment by SamvelRaja** [comment]
@czardoz addressed the comments.

**Pull request merged**